### ISSUE TITLE
multitask_wlt

### ISSUE THREAD

**Issue by complexhhs** [complexhhs/complexhhs.github.io#9]
메인코드
```cpp
int main(int argc, const char* argv[]){
    // model loading
    torch::Device device(torch::kCPU);
    cout << device <<"\n" << endl;

    if (argc != 7){
        cerr << "usage: main_torch <path-to-exported-script-module> <W> <L> <T> <Vds> <Vgs> \n";
        return -1;
    }

    torch::jit::script::Module module;
    try{
        // Deserialization from Script Module
        module = torch::jit::load(argv[1]);
    }
    catch (const c10::Error& e){
        cerr << "error loading the model\n";
        return -1;
    }

    //cout << "ok\n"; // Ok message will pop up when model is loaded succesfully!

    /*forward propagation and backpropagation */
    float W = stof(argv[2]);
    float L = stof(argv[3]);
    float T = stof(argv[4]);
    float vds = stof(argv[5]);
    float vgs = stof(argv[6]);
    bool nmos = true;

    vector<torch::jit::IValue> inputs;
    auto normal_data = input_data(vds,vgs,W,L,T); // normalizing과정 첨부!
    auto x = torch::tensor({{normal_data[0],normal_data[1],normal_data[2],normal_data[3],normal_data[4]}},device);
    cout << "x: " << x << endl;
    x.requires_grad_(true);
    inputs.push_back(x);

    auto output = module.forward(inputs).toTensor(); // neural network forward
    auto post_output = post_process(output,normal_data[0],W,L,nmos); // Vds만 normalizing, {W, L}은 원래형태 그대로
    cout << "y = " << post_output << endl;
    return 0;
}
```
- 모델로 입력시킬땐 5개의 입력변수가 <Vd, Vg, W, L, T> 순서대로 들어가야함
  - 여기서 입력변수들은 normalizing과정이 들어가야함 -> input_data함수로 변형시켰음!
  - post_process의 경우 Vds는 normalizing된 Vds, W과 L은 원래 값 그대로(normalizing과정 없이!)

normalizing 함수
```cpp
vector<float> input_data(
    float vds,
    float vgs,
    float w,
    float l,
    float t
){
    float w_max = 0.1;
    float l_max = 0.16666666666666666;
    float t_max = 60;
    float vds_max = 30;
    float vgs_max = 20;
    float vds_out;
    float vgs_out;
    float w_out;
    float l_out;
    float t_out;
    vds_out = vds/vds_max;
    vgs_out = vgs/vgs_max;
    w_out = 1./w/w_max;
    l_out = 1./l/l_max;
    t_out = t/t_max;
    vector<float> data {vds_out, vgs_out, w_out, l_out, t_out};
    return data;
}
```
- 각각의 상수들은 nmos_wlt_hyperparameter.txt에 저장되어있음
  - vds, vgs, w, l, t 각각 출력될때는 위 함수의 방식대로 normalizing되어 output됨

post_processing은 특별히 언급x
